src/patternfly/components/Skeleton/skeleton.scss

```
@@ -13,9 +13,9 @@
 
   // After
   --#{$skeleton}--after--LinearGradientAngle: 90deg;
-  --#{$skeleton}--after--LinearGradientColorStop1: var(--pf-t--global--background--color--secondary--default);
-  --#{$skeleton}--after--LinearGradientColorStop2: var(--pf-t--global--background--color--secondary--hover); 
-  --#{$skeleton}--after--LinearGradientColorStop3: var(--pf-t--global--background--color--secondary--default);
+  --#{$skeleton}--after--LinearGradientColorStop1: transparent;
+  --#{$skeleton}--after--LinearGradientColorStop2: var(--pf-t--global--background--color--loading--skeleton--default);
```

Stylelint (17.2.0)

[error] Expected custom property name "--pf-t--global--background--color--loading--skeleton--default" to be kebab-case (custom-property-pattern)

(custom-property-pattern)